db/db_impl/db_impl_write.cc

```diff
@@ -190,11 +190,34 @@ Status DBImpl::WriteWithCallback(const WriteOptions& write_options,
   return s;
 }
 
-Status DBImpl::IngestWBWI(std::shared_ptr<WriteBatchWithIndex> wbwi,
-                          const WBWIMemTable::SeqnoRange& assigned_seqno,
-                          uint64_t min_prep_log,
-                          SequenceNumber last_seqno_after_ingest,
-                          bool memtable_updated, bool ignore_missing_cf) {
+Status DBImpl::IngestWriteBatchWithIndex(
+    const WriteOptions& write_options,
+    std::shared_ptr<WriteBatchWithIndex> wbwi) {
+  if (!wbwi) {
```

Comment:
nit: can we sanity check wbwi->GetWriteBatch()->Count() !=0 and fast return?

Reply:
There is a check inside WriteImpl() for this. I assume this is unlikely to happen so doing the check later is fine.